tools/uploader-function/README.md

```diff
@@ -0,0 +1,25 @@
+# Uploader Function
+
+This solution uses the Azure Function framework to allow uploads to a DICOM Service.
+
+Steps:
+## Create app service plan
+```
```

Comment:
We talked about some of this on Friday, but here it would be good to call out detail about these parameters.

For this one, I think the resource group needs to be the same one running the DICOM service.